package.json

```diff
@@ -16,36 +16,37 @@
   "author": "Sean Price <seanprice@sevenhillstechnology.com>",
   "license": "MPL-2.0",
   "dependencies": {
-    "@fortawesome/fontawesome-free": "^5.12.0",
-    "react": "^16.9.0",
-    "react-bootstrap": "^1.0.0"
+    "@fortawesome/fontawesome-free": "^6.2.0",
+    "react-bootstrap": "^2.6.0"
   },
   "peerDependencies": {
-    "react-redux": "^7.1.1"
+    "react": "^18.2.0",
```

Comment:
Why did react get moved to a peerDepenencies?

Reply:
Libraries, as opposed to actual react apps, generally aren't supposed to declare a direct dependency on react. This can lead to more than one instance of react existing in the same app which causes a lot of problems. It's encouraged to instead put react in both `peerDependencies` and `devDependencies` -- see https://stackoverflow.com/a/30454133.